lisa/tools/mount.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from pathlib import PurePosixPath
+from typing import Any, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+
+class Mount(Tool):
+    __UMOUNT_ERROR_PATTERN = re.compile(r".*mountpoint not found", re.MULTILINE)
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def operations(
+        self, disk: str, point: str, command: str = "mount", erase: bool = True
+    ) -> None:
+        if "mount" == command:
+            self._command = "mount"
+            self.node.shell.mkdir(PurePosixPath(point), exist_ok=True)
+            cmd_result = self.run(
+                f"{disk} {point}", shell=True, sudo=True, force_run=True
+            )
+            if 0 != cmd_result.exit_code:
+                raise LisaException(f"Fail to run {self._command} {disk} {point}.")
+        elif "umount" == command:
+            self._command = "umount"
+            cmd_result = self.run(f"{point}", shell=True, sudo=True, force_run=True)
+            if erase:
+                self.node.execute(
+                    f"(echo d; echo w) | fdisk {disk}", shell=True, sudo=True
+                )
+                self.node.execute(f"rm -rf {point}", shell=True, sudo=True)
```

Comment:
The `-rf` is very dangerous. It doesn't need to use `-r` for a mount point. The mount point should be a single folder.

Reply:
without -r, can't delete folder

Reply:
I mean, use `-r` only, no `-f`.